### PR TITLE
PROGRAMMES-6402 Adding Clip Related Topics

### PR DESCRIPTION
**Pull request by votemike**
https://jira.dev.bbc.co.uk/browse/PROGRAMMES-6402

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: src/Ds2013/Presenters/Section/RelatedTopics/related_topics.html.twig [hunk 1]
```diff
@@ -1,7 +1,11 @@
 <ol class="related-topics">
     {% for relatedTopic in related_topics.getRelatedTopics() %}
         <li>
-            <a class="br-linkinvert" href="{{ path('topics_topic', {topic: relatedTopic.getId()}) }}" data-linktrack="episode_topic">{{ relatedTopic.getTitle() }} ({{ relatedTopic.getProgrammeItemCount() }})</a>
+            {% if related_topics.getContextPid() %}
+                <a class="br-linkinvert" href="{{ path('programme_topics_topic', {pid: related_topics.getContextPid(), topic: relatedTopic.getId()}) }}" data-linktrack="episode_topic">{{ relatedTopic.getTitle() }} ({{ relatedTopic.getProgrammeItemCount() }})</a>
```
Comment (votemike) [inline src/Ds2013/Presenters/Section/RelatedTopics/related_topics.html.twig]: For some reason `data-linktrack="episode_topic"` has been hardcoded in to the generic related_topics template. I can't see the attribute in LIVE. Do we actually need to keep it? Or can I delete it? If we need to keep it, should the one for clips be `data-linktrack="clip_topic"`?

Comment (votemike) [reply src/Ds2013/Presenters/Section/RelatedTopics/related_topics.html.twig]: It seems it was done as part of https://jira.dev.bbc.co.uk/browse/PROGRAMMES-6282 by @benolot and requested by @claudioneves . Do either of you know what the case should be for clips?
Was this tracking added as part of a new request over what currently is happening on LIVE?

Comment (votemike) [reply src/Ds2013/Presenters/Section/RelatedTopics/related_topics.html.twig]: Found a clip version of the ticket: https://jira.dev.bbc.co.uk/browse/PROGRAMMES-6390

**Pull request merged**